cilium-ubuntu.json

```diff
@@ -109,10 +111,16 @@
   ],
   "post-processors": [[
     {
-        "output": "cilium-ginkgo-{{.BuildName}}.box",
+        "output": "cilium-ginkgo-ubuntu-{{.BuildName}}.box",
         "type": "vagrant",
         "compression_level": 9,
         "keep_input_artifact": true
+    },
+    {
+        "type": "vagrant-cloud",
+        "box_tag": "cilium/ubuntu",
+        "access_token": "{{user `cloud_token`}}",
+        "version": "{{ user `BUILD_ID` }}"
```

Comment:
Where is this variable set? Would be nice if we could follow the x.y.z versioning scheme that Vagrant desires: https://www.vagrantup.com/docs/vagrant-cloud/boxes/catalog.html

Reply:
The BUILD_ID is the jenkins build ID number. So we can use `1.${BUILD_ID}` or `2.{BUILD_ID}` or we can use only one number (The BUILD_ID) and we're making sure that it's updating correctly.

Reply:
OK, thanks. Is there any possibility of using the x.y.z versioning scheme? Would be nice to follow the convention of VagrantCloud. But, if we have concurrent builds going at once for the VMs, might introduce versioning conflicts. Uploading to VagrantCloud could be enforced to be done  in serial order by using Jenkins' 'lockable resource' plugin  - https://wiki.jenkins-ci.org/display/JENKINS/Lockable+Resources+Plugin to ensure that only one upload is done at a time. This way we can ensure that only one version is being uploaded at a time to ensure that two builds don't upload the same version at the same time. This could be done as part of a later PR. What do you think?

Reply:
Hi @ianvernon 

- Only in master can be uploaded. And it's only one concurrent job, so build will be ok. 
- Hopefully this repo will have one or two commits per month, so I think that it's not a good idea to spend more time that necessary IMHO